wgsl/index.bs

```diff
@@ -4081,6 +4081,72 @@ TODO: user data inputs of fragment stage must be subset of user data outputs of
 
 ### Input-output matching rules TODO ### {#input-output-matching}
 
+# Language extensions # {#language-extensions}
+
+The [SHORTNAME] language is expected to evolve over time.
+
+An <dfn noexport>extension</dfn> is a named grouping for a coherent
+set of modifications to a particular version of the [SHORTNAME] specification, consisting of any combination of:
+* Addition of new concepts and behaviours via new syntax, including:
+    * declarations, statements, attributes, and built-in functions.
+* Removal of restrictions in the current specification.
+* Syntax for reducing the set of permissible behaviours.
+* A description of how the feature interacts with the existing specification, and optionally with other extensions.
+
+For example, extensions could be used to:
```

Comment:
I'd be slightly worried this will be interpreted as "here the first extensions that will be added". I'd suggest not including examples.

Reply:
I think the examples are useful. I'd rather weaken the framing.

Maybe

> Hypothetically, extensions could be used to: